jobs/datadog-route-emitter/templates/http_check.yaml.erb

```diff
@@ -0,0 +1,11 @@
+init_config: {}
+
+instances:
+  - name: route_emitter debug endpoint
+    url: http://localhost:<%= p('diego.route_emitter.debug_addr').split(':')[1] %>/debug/pprof/cmdline
```

Comment:
Yes, exactly, it is the property here: https://github.com/cloudfoundry/diego-release/blob/develop/jobs/route_emitter/spec#L26-L28

I will update the commit message to explain that.